qurator/eynollah/utils/pil_cv2.py

```diff
@@ -15,10 +15,12 @@ def pil2cv(img):
     return cvtColor(pil_as_np_array, color_conversion)
 
 def check_dpi(image_filename):
-    exif = OcrdExif(Image.open(image_filename))
-    print(exif.to_xml())
-    resolution = exif.resolution
-    if exif.resolutionUnit == 'cm':
-        resolution /= 2.54
-    return int(resolution)
+    try:
+        exif = OcrdExif(Image.open(image_filename))
+        resolution = exif.resolution
```

Comment:
This was under the assumption that https://github.com/OCR-D/core/pull/676 was merged into core.

Reply:
I don't understand. That would probably increase metadata coverage, but AFAICS nothing would change in case simply no DPI was annotated.

Reply:
I mean that this can only be triggered by `identify` not being installed (with OCR-D/core#676) or PIL not being able to open the file. I'm fixing it.

Reply:
Oh right, I forgot. So `identify` does _not_ give us 1 as fall-back, but 72. That's actually a weakness IMO (because 72 is still somewhat plausible). How do we know then when to trigger our own fall-back or heuristics?

Reply:
> How do we know then when to trigger our own fall-back or heuristics?

That is one of the reasons why this is such an annoying problem :) But probably best discussed in the core PR. For now, the `resolution == 1` check is correct but we need to revisit this here once we've agreed on how to proceed in core. The current PIL-based calculation is limiting eynollah because so much rests on "correct" DPI.